webapp/packages/core-blocks/src/useAppLoadingScreen.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
+
+export function useAppLoadingSreen() {
+  useEffect(() => {
+    const appLoadingScreen = document.getElementById('app-loading-screen');
+    if (!appLoadingScreen) {
+      return;
```

Comment:
better to log error or throw an error, so we will know why loader was not shown/hiden